cliff_adaptive_table/filter_data.py

```diff
@@ -63,7 +63,11 @@
                 'columns-v=name - Displays all columns except those whose headers match \'name\'.',
                 'columns-v=name|id or \'columns-v=name columns-v=id\' (In contrast to grep-v=<pattern>) - Displays all columns except those whose headers match either \'name\' or \'id\'.',
             ]
-        }
+        },
+        {
+            'modifier': 'sort=<column name>',
+            'description': 'Sorts rows according to the values in a specific column. Note: column name must be exact.',
```

Comment:
in which order ? can you sort the other way around ?

Reply:
added order